sov-modules/sov-modules-api/src/lib.rs

```diff
@@ -31,12 +31,19 @@ impl Prefix {
 impl From<Prefix> for sov_state::Prefix {
     fn from(prefix: Prefix) -> Self {
         let mut combined_prefix = Vec::with_capacity(
-            prefix.module_path.len() + prefix.module_name.len() + prefix.storage_name.len(),
+            prefix.module_path.len() + prefix.module_name.len() + prefix.storage_name.len() + 3,
```

Comment:
nit: magic number. Maybe add a `const DOMAIN_SEPARATOR: [u8] = [47]` and then say "+ DOMAIN_SEPARATOR.len() * 3`?

Reply:
done